csharp/ql/test/library-tests/dataflow/collections/CollectionFlow.expected

```diff
@@ -24,6 +24,7 @@ models
 | 23 | Summary: System; Span<T>; false; Span; (T); ; Argument[0]; Argument[this].Element; value; manual |
 | 24 | Summary: System; Span<T>; false; Span; (T[]); ; Argument[0].Element; Argument[this].Element; value; manual |
 | 25 | Summary: System; Span<T>; false; ToArray; (); ; Argument[this].Element; ReturnValue.Element; value; manual |
+| 26 | Summary: System.Collections.Generic; List<T>+Enumerator; false; get_Current; (); ; Argument[this].Property[System.Collections.Generic.List`1+Enumerator.Current]; ReturnValue; value; dfc-generated |
```

Comment:
This looks like a summary we shouldn't generate: It is always the case that `get_Foo` induces a readstep from `Argument[this]` to `ReturnValue`.

Reply:
I think that when we discussed it last (when you made https://github.com/github/codeql/pull/16088) - we agreed to keep model generation for properties that wasn't read/write (in this case the property only has a getter).
Do you think we should revisit this decision?

Reply:
I think that if we generate a summary for a getter that is saying exactly that it is a getter, then there is no point (since we already have a generic read step for that).

Reply:
This is also not something that we generally do.
The example in question is because there are two property implementations for `Current`, where the concrete interface implementation returns the value of the other `Current` property. The other "known" example where this can happen is for recursive property definitions (which do exist - but I sure hope that this is not a general programming pattern).

Reply:
Makes sense, thanks a lot for the explanation.